cores.json

```diff
@@ -0,0 +1,466 @@
+        "license": "https://github.com/EmulatorJS/beetle-pce-libretro/blob/master/COPYING",
+        "repo": "https://github.com/EmulatorJS/beetle-pce-libretro"
+    },
+    {
+        "name": "beetle-pcfx",
+        "makeoptions": {
+            "arguments": [],
+            "buildpath": "./",
+            "makescript": "Makefile"
+        },
+        "options": {},
+        "license": "https://github.com/EmulatorJS/beetle-pcfx-libretro/blob/master/COPYING",
+        "repo": "https://github.com/EmulatorJS/beetle-pcfx-libretro"
+    },
+    {
+        "name": "beetle-ngp",
+        "makeoptions": {
+            "arguments": [],
+            "buildpath": "./",
+            "makescript": "Makefile"
+        },
+        "options": {},
+        "license": "https://github.com/EmulatorJS/beetle-ngp-libretro/blob/master/COPYING",
+        "repo": "https://github.com/EmulatorJS/beetle-ngp-libretro"
+    },
+    {
+        "name": "beetle-wswan",
+        "makeoptions": {
+            "arguments": [],
+            "buildpath": "./",
+            "makescript": "Makefile"
+        },
+        "options": {},
+        "license": "https://github.com/EmulatorJS/beetle-wswan-libretro/blob/master/COPYING",
+        "repo": "https://github.com/EmulatorJS/beetle-wswan-libretro"
+    },
+    {
+        "name": "gearcoleco",
+        "makeoptions": {
+            "arguments": [],
+            "buildpath": "./platforms/libretro/",
+            "makescript": "Makefile"
+        },
+        "options": {},
+        "license": "https://github.com/EmulatorJS/Gearcoleco/blob/master/LICENSE",
+        "repo": "https://github.com/EmulatorJS/Gearcoleco"
+    },
+    {
+        "name": "parallel-n64",
+        "makeoptions": {
+            "arguments": [],
+            "buildpath": "./",
+            "makescript": "Makefile"
+        },
+        "options": {},
+        "license": "",
+        "repo": "https://github.com/EmulatorJS/parallel-n64"
+    },
+    {
+        "name": "mame2003-plus",
+        "makeoptions": {
+            "arguments": [],
+            "buildpath": "./",
+            "makescript": "Makefile"
+        },
+        "options": {},
+        "license": "https://github.com/EmulatorJS/mame2003-plus-libretro/blob/master/LICENSE.md",
+        "repo": "https://github.com/EmulatorJS/mame2003-plus-libretro"
+    },
+    {
+        "name": "puae",
+        "makeoptions": {
+            "arguments": [],
+            "buildpath": "./",
+            "makescript": "Makefile"
+        },
+        "options": {},
+        "license": "https://github.com/EmulatorJS/libretro-uae/blob/master/COPYING",
+        "repo": "https://github.com/EmulatorJS/libretro-uae"
+    },
+    {
+        "name": "smsplus-gx",
+        "makeoptions": {
+            "arguments": [],
+            "buildpath": "./",
+            "makescript": "Makefile.libretro"
+        },
+        "options": {},
+        "license": "",
+        "repo": "https://github.com/EmulatorJS/smsplus-gx"
+    }
+]
```

Comment:
(newline?)

Reply:
It shouldn't make a difference, but happy to standardise.

Reply:
Yeah it wouldn't make a difference. I just try to keep the last line a newline